tests/unit/test_zos_console.py

```diff
@@ -19,4 +20,11 @@ def setUp(self):
     def test_object_should_be_instance_of_class(self):
         """Created object should be instance of Console class."""
         console = Console(self.session_details)
-        self.assertIsInstance(console, Console)
\ No newline at end of file
+        self.assertIsInstance(console, Console)
+
+    @mock.patch('requests.Session.send')
+    def test_get_response_should_return_messages(self, mock_send_request):
```

Comment:
@zFernand0 added the unit tests. Is there anything which is left?

Reply:
nothing I can think of
LGTM! 😋